cmd/state-installer/cmd.go

```diff
@@ -118,8 +118,6 @@ func main() {
 	an = sync.New(cfg, nil)
 	an.Event(AnalyticsFunnelCat, "start")
 
-	logging.SetupRollbarReporter(func(msg string) { an.Event("rollbar", msg) })
```

Comment:
This is only in the installer. Do we also want to disable the reporting in the state tool and state service?

Reply:
You just saved me a huge headache 👍